config.go

```diff
@@ -252,6 +252,8 @@ type config struct {
 
 	RejectPush bool `long:"rejectpush" description:"If true, lnd will not accept channel opening requests with non-zero push amounts. This should prevent accidental pushes to merchant nodes."`
 
+	StaggerInitialReconnect bool `long:"stagger-initial-reconnect" description:"If true, will apply a randomized staggering between 0s and 30s when reconnecting to persistent peers on startup. The first 10 reconnections will be attempted instantly, regardless of the flag's value"`
```

Comment:
Why is it no longer enabled by default? With `numInstantInitReconnect` you could even say that this flag is not needed anymore.

Reply:
decided to make the behavior opt-in given that it's close to the release, decision was to make it default in a later release if the majority of users report it useful.